modules/apps/export-import/export-import-web/src/main/java/com/liferay/exportimport/web/internal/portlet/action/ExportLayoutsMVCActionCommand.java

```diff
@@ -78,7 +86,32 @@ protected void doProcessAction(
 		setLayoutIdMap(actionRequest);
 
 		try {
-			_exportImportService.exportLayoutsAsFileInBackground(
+			long backgroundTaskId = ParamUtil.getLong(
```

Comment:
Why did you need these extra line codes here? I think you just need to call the `getExportImportConfiguration(actionRequest)` method.